src/g_game.c

```diff
@@ -4637,7 +4637,7 @@ void G_BindGameVariables(void)
 {
   BIND_BOOL_GENERAL(raw_input, true,
     "Raw gamepad/mouse input for turning/looking (0 = Interpolate; 1 = Raw)");
-  BIND_BOOL_GENERAL(fake_longtics, false,
+  BIND_BOOL_GENERAL(fake_longtics, true,
```

Comment:
Okay, now for some tough choices. I think this feature is stable enough and will be welcomed by most users for casual recording of shorttics demos. However, I don't think speedrunners will want to use it. Also, I would like to preserve the philosophy of reasonable defaults and a simple menu layout.

So what should we do? Who is the primary audience for Woof? I think the best compromise is the following:

- Remove the temporary menu items added by this PR.
- Keep `fake_longtics` as a config only setting.
- Enable it by default. This is controversial, so it's also important to:
  - Make it very clear in the next changelog.
  - Make it very clear in the thread when posting about the next release.

@fabiangreffrath @rfomin What do you think?

Reply:
> * Remove the temporary menu items added by this PR.
> * Keep `fake_longtics` as a config only setting.
> * Enable it by default. This is controversial, so it's also important to:
>   
>   * Make it very clear in the next changelog.
>   * Make it very clear in the thread when posting about the next release.

I agree with all of these points. I really like this feature, and I don't think it's that controversial - in versions before 14 we have non-raw input interpolation by default, witch is much worse.

Reply:
I agree as well. This feature is a net win, recording demos with short tics will feel much more natural. And for people who rely on the "coarse haptics" of short tics for their recordings, there will be a config switch to turn it back on.